cpu/esp_common/esp-now/esp_now_gnrc.c

```diff
@@ -173,7 +173,7 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
         goto err;
     }
 
-    ((gnrc_netif_hdr_t *)netif_hdr->data)->if_pid = netif->pid;
```

Comment:
This is super un-scary. Do you insist on testing it? (Same for the adaptions of `nrfmin`, `xbee`, `nordic_ble`, `gomach` and `lwmac`.)

I'd say if `examples/gnrc_networking`, `examples/gnrc_networking_mac`, `tests/gnrc_udp` and the `ping6` shell commend is tested, this could be ACK'ed.

Reply:
Given, that I already found a bug when splitting out #11912 I'd say: never trust my code, even if it is straight forward ^^. I'd like to have the `ping6` adaptation out first (see #11916). If you think the rest is an insta-ACK, I'm happy to refrain from further splitting :-).